EIPS/eip-1775.md

```diff
@@ -0,0 +1,571 @@
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+Wallets developers have agreed on an HD derivation path for ethereum accounts using BIP32, BIP44, SLIP44, [ERC84](https://github.com/ethereum/EIPs/issues/84). Web3 wallets have implemented in a roughly similar way the rpc eth api. EIP 1102 introduced privacy through non automatic opt-in of a wallet account into an app increasing privacy.
+
+However several limitations remain in order to allow for proper design and UX for crypto permissioned apps.
+
+Most of GUI based current wallets don't allow to:
+* being able to automatically and effortlessly use different keys / accounts for each apps,
+* being able to sign some app's action without prompting the user with the same level of security as sending funds from their main accounts,
+* being able to use throwable keys to improve anonymity,
+* effortlessly signing transactions for an app without broadcasting these while still being able to perform other transaction signing as usual from their main accounts,
+* All this while being fully restorable using the user's mnemonic or hardware wallet and the HD Path determined uniquely by the app's ens name.
+
+We try to overcome these limitations by introducing a new account's type, app keys, made to be used along side the existing main accounts.
+
+These new app keys can permit to give more power and flexibility to the crypto apps developers. This can allow to improve a lot the UX of crypto dapps and to create new designs that were not possible before leveraging the ability to create and handle many accounts, to presign messages and broadcast them later. These features were not compatible with the level of security we were requesting for main accounts that hold most of an user's funds.
+
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+### Applications
+
+An app is a website (or other) that would like to request from a wallet to access app keys. It can be any form of cryptography/identity relying application, ethereum but not only.
+
+Once connected to a wallet, an application can request to access a set of accounts derived exclusively for that application using the hierarchical deterministic (HD) paths.
+
+### Applications' HD path
+
+Using the BIP 32 and BIP 43 standards, we propose to use the following HD path for each app keys:
```

Comment:
Inconsistent use of `BIP 32` (with space) and `BIP32` (without space) an `BIP-32` (with dash) throughout the document. Could you stick to a single convention please? Preferably the way it's use in the BIP repository.